src/components/form/HdForm.vue

```diff
@@ -24,15 +24,32 @@ export default {
       type: Boolean,
       default: false,
     },
+    hasChanged: {
+      type: Boolean,
+      default: null,
```

Comment:
If it is `Boolean`, I would set the default value to `true` or `false`. Do you need `null` somewhere in the code (I didn't find it)?